packages/desktop-client/src/components/Modals.tsx

```diff
@@ -230,8 +218,6 @@ function Modals({
               key={name}
               modalProps={modalProps}
               month={options.month}
-              actions={actions}
-              isGoalTemplatesEnabled={isGoalTemplatesEnabled}
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏oops?